server.js

```diff
@@ -189,7 +191,7 @@ const hbs = exphbs.create({
   extname: '.hbs',
   layoutsDir: path.join(__dirname, '/views/layouts'),
   defaultLayout: 'default',
-  partialsDir: path.join(__dirname, '/views/partials'),
+  partialsDir: [path.join(__dirname, '/views/layouts'), path.join(__dirname, '/views/partials')],
```

Comment:
question (non-blocking): why do we need the layouts in the `partialsDir`? (We already have it in `layoutsDir`) Is there something in `layouts/` that should be moved to `partials/`?

Reply:
This is primarily for QA testing.  The QA mockup (which includes email send form) was the layout and the monthly email layout was the partial in this case, with a 2-level-deep partial (Inception).  We could revert this if you like when we remove the QA mockup, though it's possible we may need to use this functionality:

The current l10n mockups are not truly loading the actual email layouts and associated styles – the html and style are actually duplicated/copied over and in some cases vary quite a bit from the actual emails.  It might be better to DRY it up and test the actual email.